tests/unit/amo/components/TestAddon.js

```diff
@@ -255,6 +255,28 @@ describe('Addon', () => {
     expect(root.find(NotFound)).toHaveLength(1);
   });
 
+  it('renders NotFound page for unauthorized add-on - 401 error', () => {
+    const id = 'error-handler-id';
+    const { store } = createStore();
```

Comment:
You should use `dispatchClientMetadata()` here to get a store as usually initialised by the app instead. It's in `tests/unit/amo/helpers`. But you can use it in the exact same way as `createStore()`, so: `const { store } = dispatchClientMetadata();` will work 😄